src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/windows-linux-hybrid-cluster.mdx

```diff
@@ -28,9 +32,9 @@ Make sure you've completed the following before going further:
 
 ## Install the current chart bundle [#install-chart-bundle]
 
-The `newrelic/nri-bundle` chart is the regular one with all the latest updates for everything on Linux nodes. You can configure the `newrelic-logging` chart to install to Windows. This feature is reliable and supported in the logging chart.
+The `newrelic/nri-bundle` chart contains the latest updates for Linux-based nodes. The `newrelic-logging` chart can also be configured during this step to install to both Linux and Windows nodes. This feature for `newrelic-logging` is reliable and supported.
 
-You can use other charts, but you must configure a `NodeSelector` to avoid scheduling to Windows. If an installation fails on a mixed cluster using default values from guided install, add the `nodeSelector` values.
+Other sub-charts in `nri-bundle` can be installed, but you must configure a `NodeSelector` to avoid scheduling to Windows. If an installation fails on a mixed cluster using default values from guided install, add the `nodeSelector` values.
 
 Create the values file with your favorite text editor, let's call it `values.yaml`. You can download a starter `values.yaml` file from the New Relic UI. Add the Kubernetes integration and choose **Helm** as the instrumentation method.
```

Comment:
Are these last few steps mixing UI install procedures with manual install? We should focus this doc on manually installing this integration rather than throwing in UI install options, as it can lead to some confusion. It definitely confused me on a few rereads.

Reply:
@akristen 

> Are these last few steps mixing UI install procedures with manual install?

Not necessarily.  One place to get an example `values.yaml` file is from our UI.  The other is to get it from our Github repo, or potentially, this docs page.